controls/os_spec.rb

```diff
@@ -282,3 +290,99 @@
     end
   end
 end
+
+control 'os-14' do
+  impact 1.0
+  title 'All password change dates are in the past'
+  desc 'The password change date is used to detect expired passwords. Entering future dates might circumvent that.'
```

Comment:
It would be great to add a reference to read more about the reasoning

Reply:
The internal document I have doesn't state much more information about this rule, that I wrote here. It is based on CIS, linking to it is sadly to so easy. I could drop this rule if the other rules are ok for you?